lib/ukargparse/argparse.c

```diff
@@ -193,3 +193,49 @@ char *uk_nextarg(char **argptr, int separator)
 	arg[arglen] = '\0';
 	return arg;
 }
+
+__ssz uk_strnkeycmp(const char *str, __sz strlen, const char *key,
```

Comment:
I know it's not a problem here, but can we pick another variable name, something other than `strlen`. :grin:

Reply:
I named it this way to point out that `strlen` is the len of the `str` parameter. The function name starts also with `strn*` to give analogy to `strncpy()` and co.